frontend/dockerfile/parser/split_command.go

```diff
@@ -23,7 +22,7 @@ func splitCommand(line string) (string, []string, string, error) {
 		}
 	}
 
-	return cmd, flags, strings.TrimSpace(args), nil
+	return cmdline[0], flags, strings.TrimSpace(args), nil
```

Comment:
Previously parser changed the casing of what the user had typed. This keeps it so that correct values can be shown on errors and should normalize to lowercase where it is actually needed.